tests/grpc/test_utils.go

```diff
@@ -0,0 +1,75 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	pGRPC "get.porter.sh/porter/gen/proto/go/porterapis/porter/v1alpha1"
```

Comment:
```suggestion
	pGRPCV1Alpha1 "get.porter.sh/porter/gen/proto/go/porterapis/porter/v1alpha1"
```

... or something else to indicate the version, since we may interact with others in the future